xplique/attributions/base.py

```diff
@@ -105,13 +117,16 @@ class WhiteBoxExplainer(BlackBoxExplainer, ABC):
     Parameters
     ----------
     model
-        Model used for computing explanations.
+        The model from which we want to obtain explanations
     output_layer
-        Layer to target for the output (e.g logits or after softmax), if int, will be be interpreted
-        as layer index, if string will look for the layer name. Default to the last layer, it is
-        recommended to use the layer before Softmax.
+        Layer to target for the outputs (e.g logits or after softmax).
+        If an `int` is provided it will be interpreted as a layer index.
+        If a `string` is provided it will look for the layer name.
+        <br/>
```

Comment:
Comments are made to explain, not to put more code. Is the <br> on purpose ?

Reply:
For my part, I am really against the idea of putting links to the doc in the code

Reply:
It is just prettier when the actual doc is generated on the website and make it more readable. It understands HTML syntax but not markdown (you can try if you want). If it is just for the <br> I would not change it (but other complicated HTML syntax yes). But if in your opinion it is really important to not have it I can delete that!